__e2e__/commands/fetchtask.test.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+const path = require('path');
+const execa = require('execa');
+const test = require('ava');
+const fs = require('fs');
+
+const rootCommand = path.join(process.cwd(), 'bin/index.js');
+const configFilePath = path.join(process.cwd(), 'config.json');
+
+const learningTracksInfo = [
+  { trackName: 'Python', fileExtension: 'py' },
+  { trackName: 'JavaScript', fileExtension: 'js' },
+];
+
+const userDefaultConfig = {
+  learningTrack: '',
+  userName: 'testConfig',
+  taskCount: 0,
+  keys: [],
+  userSubmittedFiles: [],
+};
+
+const createUserConfig = (
+  trackName,
+  fileExtension,
+  keysNumber,
+  filesNumber,
+) => {
+  let userConfig = userDefaultConfig;
+  userConfig['learningTrack'] = trackName;
+  for (let i = 0; i < keysNumber; i++) {
+    userConfig['keys'].push('testKey' + (i + 1));
+  }
+  for (let i = 0; i < filesNumber; i++) {
+    userConfig['userSubmittedFiles'].push(
+      'task' + (i + 1) + '.' + fileExtension,
+    );
+  }
+  userConfig['taskCount'] = filesNumber;
+
+  return userConfig;
+};
+
+test.serial('no config file for fetchtask', async t => {
+  if (fs.existsSync(configFilePath)) fs.unlinkSync(configFilePath);
+  const { stdout } = await execa(rootCommand, ['fetchtask'], {
+    reject: false,
+  });
+  t.snapshot(stdout);
+});
+
+test.serial('incorrect key for fetchtask', async t => {
+  let snap = null;
+  t.assert(learningTracksInfo.length > 0);
+  for (let index in learningTracksInfo) {
+    const learningTrack = learningTracksInfo[index];
+    const trackName = learningTrack['trackName'];
+    const fileExtension = learningTrack['fileExtension'];
+    const userConfig = createUserConfig(trackName, fileExtension, 30, 30);
+    fs.writeFileSync(configFilePath, JSON.stringify(userConfig));
+    const { stdout } = await execa(
+      rootCommand,
+      ['fetchtask', 'incorrectTestKey'],
+      { reject: false },
+    );
+    fs.unlinkSync(configFilePath);
+    if (snap) {
+      t.true(snap === stdout);
+    } else {
+      snap = stdout;
+    }
```

Comment:
Can you elaborate?

Reply:
Both snaps are same, so there is no need to store different snapshots. So here i just confirmed that the respective snaps are same and they match the previous snap